sandbox-backend/src/main/java/com/exadel/sandbox/team5/service/impl/CompanyServiceImpl.java

```diff
@@ -5,20 +5,32 @@
 import com.exadel.sandbox.team5.entity.Company;
 import com.exadel.sandbox.team5.mapper.MapperConverter;
 import com.exadel.sandbox.team5.service.CompanyService;
+import com.exadel.sandbox.team5.service.convertor.CSVConvertor;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
+import java.io.ByteArrayInputStream;
 import java.util.List;
 
 @Transactional
 @Service
 public class CompanyServiceImpl extends CRUDServiceDtoImpl<CompanyDAO, Company, CompanyDto> implements CompanyService {
-    public CompanyServiceImpl(CompanyDAO entityDao, MapperConverter mapper) {
+
+    private final OrderServiceImpl orderService;
+
+    public CompanyServiceImpl(CompanyDAO entityDao, MapperConverter mapper, OrderServiceImpl orderService) {
         super(entityDao, Company.class, CompanyDto.class, mapper);
+        this.orderService = orderService;
     }
 
     @Override
     public List<CompanyDto> getCompaniesByLocation(Long id) {
         return mapper.mapAll(entityDao.findAllByCountryId(id), CompanyDto.class);
     }
+
+    @Override
+    public ByteArrayInputStream getStatisticCSVFileOrdersByCompanies() {
```

Comment:
I would suggest to have following hierarchy:
```
IExportService (interface)
|-> ExportServiceCSV
|-> ExportServiceXLSX 
... etc
``` 

And move all method for exports into this interface with more clear names:
getStatisticCSVFileOrdersByCompanies -> ordersByCompanies

Also use more generic return types (Liskov principle https://en.wikipedia.org/wiki/Liskov_substitution_principle):
ByteArrayInputStream -> InputStream

Reply:
Done.